lua/compe/lazy.lua

```diff
@@ -0,0 +1,109 @@
+local Lazy = {}
+
+Lazy._sources = {
```

Comment:
Contains only builtin sources. If someone installs a 3rd party source then the chances are he will actually want that source to be loaded anyways, so we don't have to change anything else, like exposing it in public API and letting people add their own stuff here. You just won't pay for the builtin sources you don't use.